### PR TITLE
Slog level mapping

### PR DESCRIPTION
**Pull request by roblaszczak**

### CODE REVIEW

**Review by m110 — APPROVED** [review]

File: docs/hugo_stats.json [hunk 1]
```diff
@@ -317,7 +317,6 @@
       "debugging-pubsub-tests",
       "deduplicator",
       "delay-metadata",
-      "delay-on-error",
```
Comment (ccoVeille) [inline docs/hugo_stats.json:320]: Why this change?

Comment (roblaszczak) [reply docs/hugo_stats.json:320]: this is some garbage generated by hugo, I gitignored it
